src/Kentico.Xperience.Lucene/Admin/UIPages/IndexListing.cs

```diff
@@ -69,9 +69,9 @@ public override Task ConfigurePage()
 
         PageConfiguration.ColumnConfigurations
             .AddColumn(nameof(LuceneIndexStatisticsViewModel.Name), LocalizationService.GetString("integrations.lucene.listing.columns.name"), defaultSortDirection: SortTypeEnum.Asc, searchable: true)
-            .AddColumn(nameof(LuceneIndexStatisticsViewModel.Entries), LocalizationService.GetString("integrations.lucene.listing.columns.entries"))
-            //.AddColumn(nameof(LuceneIndexStatisticsViewModel.LastBuildTimes), LocalizationService.GetString("integrations.lucene.listing.columns.buildtime"))
-            .AddColumn(nameof(LuceneIndexStatisticsViewModel.UpdatedAt), LocalizationService.GetString("integrations.lucene.listing.columns.updatedat"));
+            .AddColumn(nameof(LuceneIndexStatisticsViewModel.Entries), LocalizationService.GetString("integrations.lucene.listing.columns.entries"));
+        //.AddColumn(nameof(LuceneIndexStatisticsViewModel.LastBuildTimes), LocalizationService.GetString("integrations.lucene.listing.columns.buildtime"))
+        //.AddColumn(nameof(LuceneIndexStatisticsViewModel.UpdatedAt), LocalizationService.GetString("integrations.lucene.listing.columns.updatedat"));
```

Comment:
Is this data available from Lucene? If not, can we remove this code from the repo?